typescript/sdk/src/core/adapters/CosmWasmCoreAdapter.ts

```diff
@@ -24,6 +24,11 @@ import { ChainName } from '../../types';
 
 import { ICoreAdapter } from './types';
 
+const MESSAGE_DISPATCH_EVENT_TYPE = 'wasm-mailbox_dispatch';
+const MESSAGE_DISPATCH_ID_EVENT_TYPE = 'wasm-mailbox_dispatch_id';
+const MESSAGE_ID_ATTRIBUTE_KEY = 'message_id';
+const MESSAGE_DESTINATION_ATTRIBUTE_KEY = 'destination';
```

Comment:
is there nothing useful in the `Mailbox.types` file?

Reply:
I'll check, thanks

Reply:
Nope! Worth checking though